FileMapper/blockerFunction.js

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
prefer file names in the form blocker_function.js

Reply:
Okay, will change file names once you approve the PR, so that you don't have any difficulties in reviewing.